packages/react-native-codegen-typescript-test/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@react-native/codegen-typescript-test",
+  "version": "0.0.1",
```

Comment:
Do we need this package to be published on npm?

If yes, then we need to align its version accordingly with react-native's.
If not, then please make this package private.

cc @cipolleschi @ZihanChen-MSFT